allennlp/commands/fine_tune.py

```diff
@@ -215,12 +215,20 @@ def fine_tune_model(model: Model,
     for name in tunable_parameter_names:
         logger.info(name)
 
-    trainer = Trainer.from_params(model,
-                                  serialization_dir,
-                                  iterator,
-                                  train_data,
-                                  validation_data,
-                                  trainer_params)
+    # If the config specifies a trainer subclass, we should use it.
+    # There must exist a trainer.
+    trainer_params = params.get("trainer")
+    trainer_choice = trainer_params.pop_choice("type",
+                                               Trainer.list_available(),
+                                               True)
```

Comment:
When you have hard-coded values like this, it's a lot more readable to pass them as keyword arguments.  I had to go look at the definition of `pop_choice` to remind myself of what this parameter meant, while I wouldn't have had you passed this as a keyword argument.  So `default_to_first_choice=True` would be better here.